frontend/src/components/TopBar/ApplicationMenu.tsx

```diff
@@ -114,7 +114,7 @@ export default function ApplicationMenu(): JSX.Element {
       </Dialog>
 
       <Dialog maxWidth='lg' fullWidth={true} open={changelogOpen} onClose={() => setChangelogOpen(false)}>
-        <ChangelogDialog />
+        {<ChangelogDialog />}
```

Comment:
Not sure why this has {} around it.
```suggestion
        <ChangelogDialog />
```